packages/compartment-mapper/test/fixtures-cjs-compat/node_modules/app/_generateExpected.mjs

```diff
@@ -0,0 +1,16 @@
+// run me with:
+//  node ./test/fixtures-cjs-compat/node_modules/app/test.mjs
+import * as interops from "interops"
+import * as moduleinterops from "app/moduleinterops"
+
+console.log(
+    stableOrderJson(interops)
+)
+console.log(
+    stableOrderJson(moduleinterops)
+)
+
+
+function stableOrderJson(obj) {
+    return JSON.stringify(obj, Object.keys(obj).sort());
```

Comment:
I thought so too, but it happens to sort the contents of obj.default too. Probably be because the keys are exactly the same.
My first reflex was to use substack's `json-stable-stringify` but none of the tests seem to pull in real dependencies.

Reply:
Got a decent version done in acceptable time.